data.sql

```diff
@@ -92,3 +92,41 @@ SET owner_id = (
   SELECT id FROM owners WHERE full_name = 'Dean Winchester'  
 )
 WHERE name IN ('Angemon', 'Boarmon');
+
+/* Insert data into Table vets */
+INSERT INTO  vets (name, age, date_of_graduation) VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+/* Insert data into table visits */
+INSERT INTO visits (vet_id, animal_id, date_of_visit) VALUES
+(1, 1, '2020-05-24'),
+(3, 1, '2020-07-22'),
+(4, 2, '2021-02-02'),
+(2, 3, '2020-01-05'),
+(2, 3, '2020-03-08'),
+(2, 3, '2020-05-14'),
+(3, 4, '2021-05-04'),
+(4, 5, '2021-02-24'),
+(2, 6, '2019-12-21'),
+(1, 6, '2020-08-10'),
+(2, 6, '2021-04-07'),
+(3, 7, '2019-09-29'),
+(4, 8, '2020-10-03'),
+(4, 8, '2020-11-04'),
+(2, 9, '2019-01-24'),
+(2, 9, '2019-05-15'),
+(2, 9, '2020-02-27'),
+(2, 9, '2020-08-03'),
+(3, 10, '2020-05-24'),
+(1, 10, '2021-01-11');
+
+/* Insert data into specializations table */
+INSERT INTO  specializations (vet_id, animal_id) VALUES
+(1, 1),
+(3,2),
+(3, 1),
+(4,2);
```

Comment:
🤔 I noticed a small adjustment needed in this insert query.

- [ ] In this insert query, there's an error in the relationship. Please update the query as mentioned in the previous comment.

Thank you for your effort! Making this correction will enhance the accuracy of your insert query. Keep up the good work! 👍🌟